components/form/formitems/overview.md

```diff
@@ -75,12 +76,40 @@ The `FormItem` tag exposes the following parameters which you can use to customi
 }
 ````
 
->caption The result from the code snippet above
+## Hybrid Form - Combination of manually defined and autogenerated fields
 
-![FormItem example](images/formitem-example.png)
+To combine manually defined with autogenerated fields, use an instance of the `FormAutoGeneratedItems` tag inside the `FormItems` collection.
```

Comment:
We may want to confirm explicitly that an explicit <FormItem> will not duplicate an auto-generated one. In other words, we will not generate a form item for a field that is added to an explicit <FormItem>.

Reply:
I think it's implied.